tldw_Server_API/app/core/Billing/stripe_client.py

```
@@ -0,0 +1,374 @@
+            Stripe customer ID
+        """
+        self._require_stripe()
+
+        try:
+            customer = stripe.Customer.create(
+                email=email,
+                name=name,
+                metadata=metadata or {},
+            )
+            logger.info(f"Created Stripe customer {customer.id} for {email}")
+            return customer.id
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)